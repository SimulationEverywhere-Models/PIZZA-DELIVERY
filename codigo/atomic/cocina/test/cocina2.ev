00:00:01:000 Recibe_pedido 1
00:00:02:000 Recibe_pedido 2
00:00:03:000 Recibe_pedido 3
00:00:04:000 Recibe_pedido 4
00:00:05:000 Recibe_pedido 5