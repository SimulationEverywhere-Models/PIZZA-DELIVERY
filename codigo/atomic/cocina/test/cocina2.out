00:00:05:000 ocupada_c 1
00:00:14:000 sale_pedido 1
00:00:14:000 lista_c 1
00:00:17:000 sale_pedido 4
00:00:20:000 sale_pedido 3
00:00:22:000 sale_pedido 5
00:00:23:000 sale_pedido 2
